packages/activity-monitor/src/Jellyfish/ActivityMonitor/Pm2.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Jellyfish\ActivityMonitor;
+
+use ArrayObject;
+use Generated\Transfer\Pm2\Activity;
+use Jellyfish\Process\ProcessFacadeInterface;
+use Jellyfish\Serializer\SerializerFacadeInterface;
+use RuntimeException;
+
+use function sprintf;
+
+class Pm2 implements Pm2Interface
+{
+    public const PM2_CLI = 'pm2';
+
+    public const ARGUMENT_JLIST = 'jlist';
+    public const ARGUMENT_START = 'start';
+    public const ARGUMENT_STOP = 'stop';
+    public const ARGUMENT_RESTART = 'restart';
+    public const ARGUMENT_ALL = 'all';
+
+    /**
+     * @var \Jellyfish\Process\ProcessFacadeInterface
+     */
+    protected $processFacade;
+
+    /**
+     * @var \Jellyfish\Serializer\SerializerFacadeInterface
+     */
+    protected $serializerFacade;
+
+    /**
+     * @param \Jellyfish\Process\ProcessFacadeInterface $processFacade
+     * @param \Jellyfish\Serializer\SerializerFacadeInterface $serializerFacade
+     */
+    public function __construct(
+        ProcessFacadeInterface $processFacade,
+        SerializerFacadeInterface $serializerFacade
+    ) {
+        $this->processFacade = $processFacade;
+        $this->serializerFacade = $serializerFacade;
+    }
+
+    /**
+     * @return \Generated\Transfer\Pm2\Activity[]
+     */
+    public function getActivities(): array
+    {
+        $output = $this->runCommandAsProcess([static::PM2_CLI, static::ARGUMENT_JLIST]);
+
+        $pm2Activities = $this->serializerFacade->deserialize(
+            $output,
+            sprintf('%s[]', Activity::class),
+            'json'
+        );
+
+        if (!($pm2Activities instanceof ArrayObject)) {
+            // @codeCoverageIgnoreStart
+            return [];
+            // @codeCoverageIgnoreEnd
+        }
+
+        return $pm2Activities->getArrayCopy();
+    }
+
+    /**
+     * @param int $activityId
+     *
+     * @return \Jellyfish\ActivityMonitor\Pm2Interface
+     */
+    public function startActivity(int $activityId): Pm2Interface
```

Comment:
why int in and cast to string?

Reply:
The parameter "command" of the process method "create" allows only an array of string.